bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/gemini/ghost/GhostReadoutTime.java

```diff
@@ -0,0 +1,189 @@
+/**
+ * $Id: GhostsReadoutTime.java 38186 2011-10-24 13:21:33Z swalker $
+ */
+
+package edu.gemini.spModel.gemini.ghost;
+
+import edu.gemini.spModel.config2.Config;
+import edu.gemini.spModel.gemini.ghost.GhostType.*;
+import edu.gemini.spModel.gemini.ghost.GhostType.DetectorManufacturer;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/** Maps Ghost instrument parameters to readout times */
+public class GhostReadoutTime {
```

Comment:
As we discussed, the readout time calculation is available in `edu.gemini.spModel.gemini.ghost.GhostCameras`.  Point me to where this class is used and I will see if I can help switch over to that?  It would be unfortunate to have two parallel readout time calculations to keep in sync so we want to avoid that.  On the other hand, if this is meant to be ITC-specific and not for general planned time calculations we can move it over to the ITC module.